config/istio-config/ingressgateway-fluentd-forwarder-daemonset.yaml

```diff
@@ -0,0 +1,46 @@
+#@ load("@ytt:data", "data")
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ingressgateway-fluentd-forwarder
+  namespace: #@ data.values.systemNamespace
+  labels:
+    app: ingressgateway-fluentd-forwarder
+spec:
+  selector:
+    matchLabels:
+      app: ingressgateway-fluentd-forwarder
+  template:
+    metadata:
+      labels:
+        app: ingressgateway-fluentd-forwarder
+    spec:
+      containers:
+      - name: fluentd
+        image: "logcache/cf-k8s-logging@sha256:71fd09077b70fb43dfe91e4445b6528580355003255c8860f7238aaab7b5370b"
+        resources:
+          limits:
+            memory: 200Mi
+          requests:
+            cpu: 100m
+            memory: 200Mi
+        volumeMounts:
+        - name: fluentd-config
+          mountPath: /fluentd/etc
+        - name: varlog
+          mountPath: /var/log
+        - name: dockercontainers
+          mountPath: /var/lib/docker/containers
+      terminationGracePeriodSeconds: 30
+      volumes:
+      - name: fluentd-config
+        configMap:
+          name: ingressgateway-fluentd-forwarder-config
+      - name: varlog
+        hostPath:
+          path: /var/log
+      - name: dockercontainers
```

Comment:
We've only tested this with the docker container runtime, but I'm wondering if anyone happens to know off-hand if this config is likely to work with other common Kubernetes container runtimes?